src/airport.js

```diff
@@ -4,22 +4,24 @@ const Weather = require(`./weather`)
 class Airport {
   constructor(capacity = 0) {
     this.capacity = capacity;
-    this.airportPlanes = [''];
+    this.airportPlanes = [];
   }
 
   landPlane = plane => {
-    if (this.isFull()) {
-      return "The airport is full."
-
-    } else if (this.airportPlanes.includes(plane)) {
-      return "This plane has already landed."
-
-    } else (plane instanceof Plane)
-    this.airportPlanes.push(plane)
-    return "Plane has landed!"
-
+    if (plane instanceof Plane) {
+      this.airportPlanes.push(plane);
+  
+  
+    }
   }
- 
+  
+  preventLanding = plane => {
+    if (this.isFull() && this.airportPlanes.push(plane)) {
+      return "The airport is full"
+  
+    }
```

Comment:
I would put more effort into the syntax of my code, to make it look more presentable so that it is easier to read, to understand and to test.